src/app.js

```diff
@@ -56,7 +56,25 @@ export function app(app) {
     })
   }
 
+  function hydrate(elm) {
```

Comment:
Maybe we need a vnode function that returns a vnode and share that with h. Perhaps we can save a few bytes that way and it'd better architecture-wise.